src/core_landice/Registry.xml

```diff
@@ -205,6 +205,10 @@
 		            description="x value defining region where bmlt_float_flux is applied; melt only where abs(x) is greater than xlimit."
 		            possible_values="Any positive real value"
 		/>
+		<nml_option name="config_temp_diffusive_factor" type="real" default_value="1.0e-5" units="unitless"
+		            description="the factor to estimate diffusivity (kt) in temperate ice from that in cold ice (kc), kt = f * kc"
+		            possible_values="possibly [1e-6, 1e-2]"
```

Comment:
@tongzhangice , possible_values should be something more general, e.g. "any positive real value with likely range of [1e-6, 1e-2]".

Reply:
Changed